CompanyApi/apis.http

```diff
@@ -58,4 +58,41 @@ Content-Type: application/json; charset=utf-8
 ### Case1: Sucess, 204 NO Content
 ### Case2: 404 Company NOT Found
 
-######################################################################################
\ No newline at end of file
+######################################################################################
+
+## AC6 Crete Employee By Company ID
+POST https://{{hostname}}:{{port}}/api/companies/{companyId}
+
+{
+  "Name": 
+  "CompanyId"

```

Comment:
you have contained the company id in the url, no need to have in the request body